VideoLocker/res/layout/discussion_user_profile_row.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/discussion_user_profile_row"
```

Comment:
The `discussion_` prefix seems redundant here, given that it's already present in the layout name. Same goes for the other views here, although it does seem to be the style used by the existing layouts in the module 😦